### PR TITLE
Documentation updates & bug fix 🐛🔨

### LINKED ISSUE #282: Avatar does not default back to "barb" when girl is entered
**Describe the bug**
If an invalid avatar is entered, the avatar selection does not default to an existing character 
**To Reproduce**
1. Set avatar to 'girl'
2. Run program
3. See how it searches for the 'girl' filepath

**Expected behavior**
Avatar should default back to barb

### LINKED ISSUE #284: Peer Testing Report: Brief description of your system
**Describe the solution you'd like**

- (10%) Brief description of your system and its current set of features available in the testing session

### LINKED ISSUE #287: Peer Testing Report: List of issues discovered and prioritized
**Describe the solution you'd like**

- (20%) List of issues discovered, prioritized and presented in order of high, medium, then low.

### PR DESCRIPTION
**Pull request by LoganParker**
Fixed the error with input handling, as well as completes some of the smaller sections of the report, which should be fleshed out more when integrated into the full report document - or not, I think theyre good enough but I'm unsure what our report will look like at the moment.

Removed the __length from the characters enum, as it's not used & there are other ways of reading the length if required - don't want someone passing length in as a character!

- Closes #287 
- Closes #284
- Closes #282

### CODE REVIEW

File: speaking-portal-project/src/types.ts [hunk 1]
```diff
@@ -15,7 +15,6 @@ export type Phoneme = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'X'
 export enum AvatarNames {
     barb,
     boy,
-    __LENGTH
```
Comment (LoganParker) [inline speaking-portal-project/src/types.ts:18]: This is the only important change, the rest of the changes are related to code linting

**Review by vnajack — APPROVED** [review]
It's so satisfying to see all the formatting changes that will make the code consistent. Thank you for doing that in addition to the avatar selection bug fix. I'll compile the report's markdown files tomorrow.

**Comment by edouarde1** [comment]
Looks like a conflict on this PR

File: speaking-portal-project/src/types.ts (hunk 1 above)
Comment (edouarde1) [reply speaking-portal-project/src/types.ts:18]: Great change and nice catch in peer testing

**Review by edouarde1 — APPROVED** [review]
Thanks for this bug fix, other then the conflict this looks all good

**Pull request merged**